README-CN.md

```diff
@@ -41,11 +41,25 @@
 
 [Nebula Graph Cloud Service](https://cloud.nebula-graph.com.cn/) 是 Nebula Graph 的图数据库云服务平台（Database-as-a-Service，DBaaS），支持一键部署 Nebula Graph。Nebula Graph Cloud Service 服务目前处于公测阶段。公测期间不会就云服务收取任何费用，欢迎 **免费试用**。
 
+## 关于这个 Repo
```

Comment:
Maybe move this section to the start of the topic. This should be the first thing that a reader needs to know.
So does in the English version.

Reply:
Agreed, will do so.

Reply:
done